docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md

```diff
@@ -18,6 +18,12 @@ Clients in Camunda 8 SaaS and applications in Camunda 8 Self-Managed provide a s
 
 ## Steps
 
+:::caution Keycloak URLs
+As of the 8.5 release, Identity uses the Keycloak frontend URL instead of the backend URL. This change may affect customers who have blocked the Keycloak frontend URL from other services, including Camunda applications, and can potentially impact Identity's functionality.
```

Comment:
💭 Just a consideration here, the change was made specifically in 8.5.3 so not sure if we should make this notice more specific?